### PR TITLE
[CMOS-3982]: Update Image component to handle fallback images

### PR DESCRIPTION
**Pull request by chaitany6143**
https://jira.we.co/browse/CMOS-3982

### CODE REVIEW

File: src/components/Image/Image.jsx [hunk 1]
```diff
@@ -1,23 +1,50 @@
...
+
 import {
   getDataAttrs,
   getDataProps,
 } from '../../dataUtils';
 
+const defaultImage = 'https://cdn.spacetelescope.org/archives/images/screen/s82e5407.jpg';
+
 class Image extends React.Component {
+  constructor() {
```
Comment (aaroncouture) [inline src/components/Image/Image.jsx]: I think you can set inital state directly:

state = { errored: false }

File: src/components/Image/Image.jsx [hunk 2]
```diff
@@ -1,23 +1,50 @@
...
+    super();
+    this.state = {
+      errored: false,
+    };
+  }
+
+  getImageUrl() {
```
Comment (aaroncouture) [inline src/components/Image/Image.jsx]: minor: should we reset errored: false if props.src changes?

File: src/components/Image/Image.jsx [hunk 3]
```diff
@@ -1,23 +1,50 @@
...
-import style from './style.scss';
+import cn from 'classnames';
+import styles from './style.scss';
+
 import {
   getDataAttrs,
   getDataProps,
 } from '../../dataUtils';
 
+const defaultImage = 'https://cdn.spacetelescope.org/archives/images/screen/s82e5407.jpg';
```
Comment (pseuyi) [inline src/components/Image/Image.jsx]: do we still want this default image btw? it's a photo of astronauts lol

Comment (chaitany6143) [reply src/components/Image/Image.jsx]: lol, tt's just used as a default prop for src and fallback, so kept it.

File: stories/Image.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import Image from '../src/components/Image/Image';
+
+
+storiesOf('Image', module).add('image', () => (
+  <Image />
+));
+
+storiesOf('Image', module).add('image click', () => (
```
Comment (chelseawework) [inline stories/Image.js]: Should this change the way the image looks? If not, it shouldn't be in the storybook.

File: stories/Image.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import Image from '../src/components/Image/Image';
+
+
+storiesOf('Image', module).add('image', () => (
```
Comment (chelseawework) [inline stories/Image.js]: You added Image to the storybook! Awesome! 🎉

**Review by RainbowStripes — DISMISSED** [review]
You need to add it to the storyboook index (e.g. `stories/index.js:8`).

**Comment by chelseawework** [comment]
Even though we don't "require" anything in the Plasma comments. You should probably give a brief description and link the JIRA ticket.

**Review by aaroncouture — APPROVED** [review]

**Review by kangax — APPROVED** [review]

**Pull request merged**